websites/R/Richup.io/metadata.json

```
@@ -0,0 +1,34 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.7",
+  "service": "Richup.io",
+  "author": {
+    "id": "162969778699501569",
+    "name": "EGGSY#3388"
```

Discriminator added by old version of pmd.
```suggestion
    "name": "EGGSY"
```
Also, this file is not formatted properly. Use `yarn format` to reformat it.

---

now you got 4 more files to review :>